r/README.md

```diff
@@ -102,6 +102,43 @@ elsewhere, you’ll need to build it from source too.
 
 First, install the C++ library. See the [developer
 guide](https://arrow.apache.org/docs/developers/cpp/building.html) for details.
+It's recommended to make a `build` directory inside of the `cpp` directory of
+the Arrow git repository (it is git-ignored). Assuming you are inside `cpp/build`,
+you'll first call `cmake` to configure the build and then `make install`.
```

Comment:
Are you advocatin "make"? Even on Windows?

Reply:
R development is mostly happening in a MinGW / msys2-based environment on Windows, so `make` would be a default choice.

Reply:
I guess I could change it to `cmake --build . --target install`? I'm kinda assuming that if you know how to change the cmake generator then you also know how to invoke it.

Reply:
>  I'm kinda assuming that if you know how to change the cmake generator then you also know how to invoke it.

Sounds fair.